static/app/views/performance/landing/widgets/widgets/trendsWidget.tsx

```diff
@@ -161,6 +162,7 @@ export function TrendsWidget(props: PerformanceWidgetProps) {
                   initialConditions,
                   additionalQuery: {
                     trendFunction: trendFunctionField,
+                    statsPeriod: eventView.statsPeriod || DEFAULT_STATS_PERIOD,
```

Comment:
I _think_ this should work instead of munging location, will check tomorrow before merging.